### PR TITLE
Setup/theme: Setup theme config design system using MUI

### PR DESCRIPTION
**Pull request by aatbip**
### Task/Ticket

[Setup theme config design system using MUI]

#### The main changes are

- [x] Adds theme config file using MUI
- [x] Adds MUI and Emotion dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +32,9 @@
     "postinstall": "prisma generate"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.3",
```
Comment (sazanrjb) [inline package.json:35]: what is this library about?

**Review by sazanrjb — APPROVED** [review]
Also add about the MUI setup in README

File: package.json (hunk 1 above)
Comment (aatbip) [reply package.json:35]: MUI itself depends on emotion or styled component. While installing MUI, we also install emotion or styled component depending on our preference. Both emotion and styled component are CSS in JS styling engines. For your reference, please see [here](https://mui.com/material-ui/getting-started/installation/#with-styled-components).
@sazanrjb

**Comment by aatbip** [comment]
> Also add about the MUI setup in README

Added MUI instruction in Readme. Setup details is not added because no setup is required. Everything is ready to be used. @sazanrjb

**Review by sazanrjb — APPROVED** [review]

**Pull request merged**